### PR TITLE
AB#11143 Email verificiation design changes

### PR DESCRIPTION
**Pull request by jacobtoye**
ADO card: ☑️ [AB#11143](https://dev.azure.com/S72/SHIFT72/_workitems/edit/11143)

Designs: 
https://www.figma.com/file/jVvevWJZncR2BzBtPmahdX/Modals?node-id=1409-12272&t=hkjP3ffDGnHG3FuO-0

## Description of work
These are the changes from the design review of the Email Verification work we did. See the figma file for how each screen is meant to look.

This PR also include some general changes applied to other areas, they are:

- Form buttons now span the width of their container
- Inputs no longer have a background, except if they are disabled

image

I was unable to do some of the requested changes, namely:

- Update the checkboxes to match the designs. To do this we would need to roll our own with icons for checks etc.
- Update the size of the page headers. The designs use 24 px font while the site uses 35px fonts. Design should probably do a typography audit to ensure we are using the right size/styles through out the site.

## Related PRs 

https://github.com/indiereign/shift72-web/pull/414

### Affected Clients
 - All clients who use Kibble

## Checklist
- [ ] CI tests are passing Github actions (inc. linting)
- [ ] Key areas of the feature outlined for context and testing
- [ ] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [ ] Updated changelog (if applicable)
- [ ] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/verifyemail.html.jet [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("signup_page_header"), index=false)}}
+{{end}}
+
+{{block body()}}
+  <main id="main" class="page form-page">
+    <div class="page-header">
+      <h1>Verify your email</h1>
```
Comment (PeculiarGoat) [inline site/verifyemail.html.jet:10]: All of this text needs to be translations!

File: site/verifyemail.html.jet [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("signup_page_header"), index=false)}}
```
Comment (PeculiarGoat) [inline site/verifyemail.html.jet:4]: Probably want to change that signup_page_header to something new.

File: site/verifyemail.html.jet [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{{end}}
+
+{{block body()}}
+  <main id="main" class="page form-page">
+    <div class="page-header">
+      <h1>Verify your email</h1>
+    </div>
+    <div class="page-content text-center">
```
Comment (PeculiarGoat) [inline site/verifyemail.html.jet:12]: :x: `text-center`... that's a Bootstrap class... a one line Bootstrap class with `!important` 🤮 

To be consistent with core-template, I would swap that `form-page` class on the main element (this isn't a form page) for `verifyemail-page` and add CSS for this:

```
.verifyemail-page {
  .page-content {
    text-align: center;
  }
}
```

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Some stuff to do!

File: site/verifyemail.html.jet (hunk 1 above)
Comment (jacobtoye) [reply site/verifyemail.html.jet:10]: Oh whoops I completely forgot :S

File: site/verifyemail.html.jet (hunk 2 above)
Comment (jacobtoye) [reply site/verifyemail.html.jet:4]: It's meant to be the same. When the user has the verify email toggle on then this is the first step of creating a new account.

File: site/verifyemail.html.jet (hunk 3 above)
Comment (jacobtoye) [reply site/verifyemail.html.jet:12]: Right so we are wanting to never directly use the bootstrap helper classes. I'll update to follow suit (in the interest of time I'm not going to update the other places in core-template that use `text-center`).

I'm guessing the reasoning behind using more singular classes is to support customizations easier? I.e. you only need to override the `.verifyemail-page` class?

Comment (PeculiarGoat) [reply site/verifyemail.html.jet]: Yup that's correct.

**Review by PeculiarGoat — APPROVED** [review]
Nice 👍

**Pull request merged**